### ISSUE TITLE
Better output for MiniTest::Spec failed assertions

### ISSUE THREAD

**Issue by ordinaryzelig** [turn-project/turn#55]
Because of the nature of how spec assertions are defined in MiniTest (infect_an_assertion), the output for a failed assertion looks like this:

```
  OutputSpec:
     FAIL test_0001_output (0.00s) 
          Expected 1, not 2.
          (eval):4:in `must_equal'
```

Without turn, the output looks like this:

```
    1) Failure:
  test_0001_output(OutputSpec) [turn.rb:6]:
  Expected 1, not 2.
```

The line number of the failure would be nice to have in the turn output.

(I'm using mintiest 2.5.1 and turn 0.8.2.)

**Comment by davekrupinski** [comment]
+1

**Comment by the-architect** [comment]
+1

**Comment by sunaku** [comment]
+1

**Comment by trans** [comment]
This should be resolved with the upcoming 0.9 release. I am going to go ahead and close the ticket. But please open a new one if the next release doesn't perform as needed in this regard.

Thanks.
